shared/router-v2/header/index.desktop.js

```diff
@@ -99,7 +99,7 @@ class Header extends React.PureComponent<Props, State> {
               type="iconfont-arrow-left"
               style={backArrowStyle}
               color={iconColor}
-              onClick={this.props.allowBack || this.props.loggedIn ? this.props.onPop : null}
```

Comment:
cc @cjb I removed this so the back button would be disabled at the root of the tab. Do we need this for something?

Reply:
Nope!